src/room.ts

```diff
@@ -23,7 +23,11 @@ interface Room {
     /**
      * Returns an array of events happened on the previous tick in this room.
      */
-    getEventLog(raw?: boolean): EventItem[];
+    getEventLog(raw?: false): EventItem[];
+    /**
+     * Returns an raw JSON string of events happened on the previous tick in this room.
+     */
+    getEventLog(raw: true): string;
```

Comment:
While Typescript has some limitations when dealing with overload function signatures: the built-in tool type `Parameter` and `ReturnType` only infer types from the last overload ([see playground](https://www.typescriptlang.org/play?#code/JYOwLgpgTgZghgYwgAgKIDcLgJKQLbIDeyAvgFBkAmECANnFCjAK4gJjAD2IyA5hGAxYwAGU68AFFDgB3APwAuZPFoBnCAEolQnPgDaAXQDcVGvUbLW7Lj36DM4MZOkylYKM01LV70LxNkYACeAA4o2HB4ACoeKAC8yAAKDJEC0KoAPMFhnDB8Ajqi4gB8FNR0DExWHNz59sJOAExSsm6xWsg+UH4m5eZVbDW2BQ5FvM0uispwal5oo7gQeIYB2eGRAPIhQzMAYjPqyAnJ0nhpUJlruXWFTcVAA) ), I think we do not need to be that strict. Therefore, I think we do not need to care about the order of overloads here.